pages/common/mpc.md

```diff
@@ -0,0 +1,28 @@
+# mpc
+
+> Music Player Client.
+> Program for controlling Music Player Daemon (MPD).
+
+- Toggle between play and pause:
```

Comment:
I'd say just "Toggle play/pause", but this is just a suggestion -- feel free to ignore it if you prefer the current phrasing.